src/routing_table.c

```diff
@@ -371,6 +371,15 @@ olsr_delete_rt_path(struct rt_path *rtp)
     current_inetgw = NULL;
   }
 
+  /* remove reference from rt_best */
+  struct rt_entry* rt;
+  OLSR_FOR_ALL_RT_ENTRIES(rt)
+  if(rt->rt_best == rtp){
+	  rt->rt_best = NULL;
+	  OLSR_PRINTF(7, "Removed rt_best.\n");
```

Comment:
do not use tabs on these 2 lines, indenting is wrong
exit the loop after you set  rt->rt_best = NULL;

Reply:
please check that exiting the loop is correct.
IOW: the loop should only be left when the same route is never stored in another entry.